### PR TITLE
Galaxy dev kmer read coverage

### PR DESCRIPTION
**Pull request by DLBPointon**
Addition of the kmer_read_coverage subworkflow.

File output to treeval upload as ${sample_id}_${kmer}.bigwig

### CODE REVIEW

File: assets/local_testing/nxOscSUBSET.yaml
Comment (weaglesBio) [inline assets/local_testing/nxOscSUBSET.yaml:1]: test .yaml needs updating to new property names.

File: subworkflows/local/yaml_input.nf
Comment (weaglesBio) [inline subworkflows/local/yaml_input.nf:1]: existence

**Review by weaglesBio — CHANGES_REQUESTED** [review]
Looks good, no issues aside from SUBSET.yaml needing updating and spelling.

File: subworkflows/local/yaml_input.nf
Comment (DLBPointon) [reply subworkflows/local/yaml_input.nf:1]: Always the spelling.
Thanks, done.

File: assets/local_testing/nxOscSUBSET.yaml
Comment (DLBPointon) [reply assets/local_testing/nxOscSUBSET.yaml:1]: Fixed now

**Comment by DLBPointon** [comment]
Just as a note, we now have 3 places where we are grabbing long read data and concatting it into one file. At this point it would be smart to make a "longread collector subworkflow"

**Pull request merged**